tests/local.rs

```diff
@@ -4,23 +4,33 @@ extern crate serial_test_derive;
 extern crate fantoccini;
 extern crate futures_util;
 
-use fantoccini::{error, Client, Locator};
+use hyper::client::connect::Connect;
+
+use fantoccini::{error, Locator};
+
+use fantoccini::Client;
```

Comment:
Let's put this back with the other imports further up the way it was before so we simplify the diff